src/main/resources/static/javascript/mainApp/grouping.controller.js

```diff
@@ -1744,46 +1755,29 @@
         };
 
         /**
-         * Checks the state of the main "Select All" checkboxes and tooltip based on checkbox changes.
+         * Checks the state of the main "Select All" checkbox based on checkbox toggles.
          *
          * @param {Object} item - An object containing information about the group and pagination.
```

Comment:
I would like to change the name of the `item` object to reflect a more accurate description. I'm thinking of `groupMembers` because the three properties all revolve around the members within the group.

1. `allItems` - all of the members within a group
2. `pageItems` - all of the members within the page (max of 20 per page)
3. `pageNumber` - the page number that the user currently is on (but we can take this out because it isn't being used anymore since we assume the user will see the tooltip and remove all of the members at once)

What do you think about the name change and getting rid of the `pageNumber` property?

Reply:
That sounds good.

Reply:
I think it was named item in the first place because we have a $scope variable named $scope.item, but this function does not use that $scope variable.

Reply:
Decided to leave the `pageNumber` property as it was being used in the event listener functions as well.